apps/web/components/shared/project-card-variants.tsx

```
@@ -1,384 +1,384 @@
+	Globe,
+	Heart,
+	Share2,
+	Star,
+	Trophy,
 } from 'lucide-react'
 import type React from 'react'
 import { useEffect, useState } from 'react'
-import { nftTiers, nftCollection, comments } from '../mocks/mock-data'
+import { comments, nftCollection, nftTiers } from '../mocks/mock-data'
 
 type ProjectCardProps = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)